package.json

```diff
@@ -16,10 +16,8 @@
     "build-system-uri": "cd system_uri && cargo build && cd ../ && cp -f system_uri/target/debug/*system_uri* ./src/native",
     "build-safe-lib": "cd native/safe_app && cargo build --features \"use-mock-routing testing\" && cd ../../ && cp -f native/target/debug/*safe_app* ./src/native",
     "build-libs": "npm run build-system-uri && npm run build-safe-lib",
```

Comment:
Yes, it takes long. For development it's probably enough to be able to direct the deps. downloader to grab the libraries from my local cloned repo of the safe_app/system_uri.